ui/values/localizable/fr.go

```diff
@@ -455,4 +455,5 @@ const FR = `
 "selectDexServerToOpen" = "Select the Dex server you would like to open."
 "addDexServer" = "Add dex server"
 "canBuy" = "Can Buy"
+"ok" = "OK"
 `
```

Comment:
This can also be translated into French.
Using google translate, we can use D'accord